ui/src/main/webapp/src/app/components/form.component.ts

```diff
@@ -24,15 +24,17 @@ export class FormComponent {
     handleError(error:any) {
         this.errorMessages = [];
         if (!error) {
-            this.errorMessages.push("Server call failed!");
-        } else if (error.parameterViolations) {
+            this.errorMessages.push("Server call failed.");
+        }
+        else if (error.parameterViolations) {
             error.parameterViolations.forEach(violation => {
                 console.log("Violation: " + JSON.stringify(violation));
                 this.errorMessages.push(violation.message);
             });
-        } else {
+        }
+        else {
```

Comment:
I'm not fan of those formatting changes.
I'm for keeping closing bracket and else on the same line as before (and as we have it everywhere else).